frontend/src/components/layout/CircleBackground.tsx

```diff
@@ -31,49 +33,51 @@ function CircleBackgroundLayout({ children }: MyProps) {
     <Content>
       <Header />
       <MainContainer>
-        <LandingPageContainer>
+        <CircleBackgroundContainer>
+          {children}
+        </CircleBackgroundContainer>
+        <div>
```

Comment:
I made a screencast with some of my personal design nits / preferences: [Design Nits on #118 Frontpages](https://www.loom.com/share/3e8affa5e05f43cdb67c4db248bc4acb). I think that the circle background isn't entirely aligned on the other frontpages.

Reply:
I made a slight fix to this by slightly lowering the top of the containers for the create and join pages. I don't think we'll be able to get it perfect on every screen size though, for example: 

![Small width screen with overlapping text](https://user-images.githubusercontent.com/24768574/111862199-ffdba980-8910-11eb-85a5-0630615da922.png)

Any thoughts?

Reply:
It's not perfect, but I don't have a huge problem with these designs, especially if you like this more than just the plain background on these pages. It also means less development on the likely slightly less important tasks here, which should be a good thing.

Maybe in the future, we could have this layout depend again on the central container rather than the whole page, and then each of these pages would be customized to that. If the circles / blobs moved fluidly (animation-wise) when transitioning the pages, that would be the optimal design in my opinion; however, this may be tough (or even infeasible), and I'm ok with this setup. We could look at it again in the future, but if this works by you for this PR, it's all good by me.

Also, thanks for making the buttons the same size as the input ❤️